publiq-ui/pub-pagination.stories.mdx

```diff
@@ -0,0 +1,41 @@
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs/blocks';
+import PubPagination from './pub-pagination.vue';
+
+<Meta title="pub-pagination" component={PubPagination} />
+
+export const Template = (args, { argTypes }) => {
+  return {
+    props: Object.keys(argTypes),
+    components: { PubPagination },
+    template: '<pub-pagination v-bind="$props" />',
+  };
+};
+
+export const commonArgs = {
+  rows: 50,
+  prevText: 'Previous one',
+  nextText: 'Next one',
+};
+
+export const commonArgTypes = {
+  prevText: {
+    defaultValue: "this.$t('pagination.previous')",
```

Comment:
@bertramakers these default values don't seem to come through. Although it seems to be following the [docs](https://storybook.js.org/docs/react/api/argtypes)

Reply:
it is meant just as a string, because I don't think we have the ability to use the $t function here, correct me if I'm wrong

Reply:
Indeed you can't use Nuxt functions here since Storybook is not running in Nuxt. It's just importing our ui components. It's okay to use hardcoded (untranslated) strings here IMHO though. It's purely technical documentation.

Reply:
Yes that is also my intention, but the defaultValue doesn't work in the docs part in storybook

Reply:
Additionally, you're passing a value already in `commonArgs` so it's normal the default values aren't used.

Reply:
If I set
```
export const commonArgs = {
  rows: 50,
};

export const commonArgTypes = {
  prevText: {
    defaultValue: "Previous",
  },
  nextText: {
    defaultValue: "Next",
  },
};
```

I see:
![Screenshot 2020-09-04 at 10 56 07](https://user-images.githubusercontent.com/959026/92221128-48fc6c80-ee9d-11ea-8129-e55c6c9517ac.png)

Maybe you're confusing the default value of the control, and the default value of the prop on the component itself? The latter one should be changed in the component itself.

Reply:
Ok I thought I could override the default value coming from the vue component to be a string we decided, because I would have liked to change the long string of `function() { this.$t...`

Reply:
Ah it is possible! with `table: {
      defaultValue: { summary: 'Eigen default' },
    }`

Reply:
It would be better to have a default that's not reliant on a Nuxt plugin (`nuxt-i18n`) of course. Not sure how to handle this. Maybe also something to discuss with @geoffreydhuyvetters once he feels better.

Reply:
> Ah it is possible! with `table: {
> 
>       defaultValue: { summary: 'Eigen default' },
> 
>     }`

But wouldn't it be weird to show some kind of "random" default value in Storybook while it's not actually the default? 🤔

Reply:
Ah I see you set `$t(...)` as the default now to clean it up visually. Still it raises the questions whether it makes sense to access `nuxt-i18n` in a component library. See for example https://stackoverflow.com/a/59218742